src/main/java/net/zhuoweizhang/raspberryjuice/RemoteSession.java

```diff
@@ -469,6 +610,47 @@ protected void handleCommand(String c, String[] args) {
 					send("Fail");
 				}
 				
+			// entity.getEntities
+			} else if (c.equals("entity.getEntities")) {
+				int distance = 10;
+				if (args.length > 1)
+					distance = Integer.parseInt(args[1]);
+				Entity playerEntityId = plugin.getEntity(Integer.parseInt(args[0]));
+				StringBuilder bdr = new StringBuilder();
+				for (Entity e : world.getEntities()) {
+					if ( e.getType().isSpawnable() && e.getType().getTypeId() >= 0 && getDistance(playerEntityId, e) <= distance) {
+						bdr.append(e.getEntityId());
+						bdr.append(",");
+						bdr.append(e.getType().getTypeId());
+						bdr.append(",");
+						bdr.append(e.getType().toString());
+						bdr.append(",");
+						bdr.append(e.getLocation().getX());
+						bdr.append(",");
+						bdr.append(e.getLocation().getY());
+						bdr.append(",");
+						bdr.append(e.getLocation().getZ());
+						bdr.append("|");
+					}
+				}
+				send(bdr.toString());
+					
+			// entity.removeEntityType
+			} else if (c.equals("entity.removeEntityType")) {
```

Comment:
Im not sure this method is required, it seems very specific and could be achieved using existing methods.